shell/platform/android/io/flutter/embedding/android/FlutterImageView.java

```diff
@@ -200,15 +211,12 @@ public void resizeIfNeeded(int width, int height) {
     if (width == imageReader.getWidth() && height == imageReader.getHeight()) {
       return;
     }
-    // Close resources.
-    if (nextImage != null) {
-      nextImage.close();
-      nextImage = null;
-    }
+    imageQueue.clear();
     if (currentImage != null) {
```

Comment:
This `if` can be removed now that `currentImage.close()` has been removed

Reply:
Done.